receiver/kafkametricsreceiver/metadata.yaml

```diff
@@ -16,15 +16,89 @@ attributes:
   group:
     description: The ID (string) of a consumer group
     type: string
+  broker:
+    description: The ID (integer) of a broker
+    type: int
 
 metrics:
 #  brokers scraper
   kafka.brokers:
+    enabled: true
+    description: "[DEPRECATED] Number of brokers in the cluster."
+    unit: "{brokers}"
+    gauge:
+      value_type: int
+    warnings:
+      if_enabled: The metric is deprecated and will be removed. Use `kafka.brokers.count`
+  kafka.brokers.count:
     enabled: true
```

Comment:
Both `kafka.brokers` and `kafka.brokers.count` should not be enabled. Let's make this one disabled with the 
```
warnings:
  if_enabled_not_set: This metric will be enabled by default in the next versions.
```

This operation should be applied to all the metrics that we want to make enabled eventually. Doing it right away is considered a breaking change. BTW do we want to make all of them enabled, or can some be considered optional?

See this doc for more guidelines https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/scraping-receivers.md

Reply:
disabled
Other (new metrics) I'd make enabled by default to make it simpler for users and more data but if on your side it's considered  breaking change, I could disable them.

Reply:
btw I can't reproduce the tests error

Reply:
btw I can't reproduce the tests error

Reply:
> Other (new metrics) I'd make enabled by default to make it simpler for users and more data but if on your side it's considered breaking change, I could disable them.

Those that we want to be enabled, should be disabled at the first step with the same warning, then enabled in a couple of next releases

Reply:
done

Reply:
How users should be able to enable them on their side?

Reply:
All the metrics can be enabled or disabled by the user as

```
metrics:
  kafka.brokers:
    enabled: true
```

The metrics defined in the metadata.yaml are automatically exposed as a configuration interface